infra/api/service/main.tf

```
@@ -140,7 +140,11 @@ module "service" {
     }
   } : null
 
-  extra_environment_variables = merge(local.service_config.extra_environment_variables, { "ENVIRONMENT" : var.environment_name })
+  extra_environment_variables = merge(
+    local.service_config.extra_environment_variables,
+    { "ENVIRONMENT" : var.environment_name },
+    { "DRAFT_S3_BUCKET_ARN" : aws_s3_bucket.draft_documents.arn }
```

nit

```suggestion
    { "DRAFTS_S3_BUCKET_ARN" : aws_s3_bucket.draft_documents.arn }
```

`DRAFT_S3` sort of implies that the S3 bucket itself in is draft